### PR TITLE
chore: remove unused libs, add used libs

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- remove unused libs
- add type-fest and bcrypto to deps as they are used

### CODE REVIEW

File: packages/connect-webextension/package.json [hunk 1]
```diff
@@ -48,7 +48,6 @@
         "@babel/preset-typescript": "^7.23.3",
         "@trezor/node-utils": "workspace:*",
         "@trezor/trezor-user-env-link": "workspace:*",
-        "@types/chrome": "^0.0.260",
```
Comment (mroz22) [inline packages/connect-webextension/package.json]: you sure it is not used?

Comment (mroz22) [reply packages/connect-webextension/package.json]: my vscode sources types for this line from @types/chrome https://github.com/trezor/trezor-suite/blob/develop/packages/connect-web/src/channels/serviceworker-window.ts#L15

File: packages/suite-web/package.json [hunk 2]
```diff
@@ -40,13 +40,11 @@
         "@trezor/transport": "workspace:*",
         "@trezor/trezor-user-env-link": "workspace:*",
         "@trezor/utils": "workspace:*",
-        "@types/chrome-remote-interface": "^0.31.14",
         "@types/cypress-image-snapshot": "^3.1.8",
         "@types/react": "18.2.79",
         "@types/react-dom": "18.2.19",
         "@types/react-router": "^5.1.20",
         "@types/react-router-dom": "^5.1.7",
-        "chrome-remote-interface": "^0.33.0",
```
Comment (mroz22) [inline packages/suite-web/package.json:49]: this used to be used in cypress tests. not sure if it is still active, haven't been maintaining it for a while.

File: packages/utxo-lib/package.json [hunk 3]
```diff
@@ -41,6 +41,7 @@
     "dependencies": {
         "@trezor/utils": "workspace:*",
         "bchaddrjs": "^0.5.2",
+        "bcrypto": "5.4.0",
```
Comment (mroz22) [inline packages/utxo-lib/package.json]: normally eslint or some other tools yells when a dependency is missing, why didn't it catch this? 🤔

File: packages/suite-web/e2e/cypress.config.ts [hunk 4]
```diff
@@ -1,6 +1,5 @@
 import { defineConfig } from 'cypress';
 
-// import CDP from 'chrome-remote-interface';
```
Comment (mroz22) [inline packages/suite-web/e2e/cypress.config.ts:3]: hmm lines 41 and 176 are dead too. I think this was used to add hover ability to cypress. not sure why we commented this out? maybe @andrcj knows

File: packages/connect-webextension/package.json (hunk 1 above)
Comment (tomasklim) [reply packages/connect-webextension/package.json]: returned back

File: packages/suite-web/package.json (hunk 2 above)
Comment (tomasklim) [reply packages/suite-web/package.json:49]: it was commented out so i guess it is ready to be removed

File: packages/suite-web/e2e/cypress.config.ts (hunk 4 above)
Comment (tomasklim) [reply packages/suite-web/e2e/cypress.config.ts:3]: not sure if 176 is dead, `resetCRI` is used in `commands.ts`

File: packages/utxo-lib/package.json (hunk 3 above)
Comment (tomasklim) [reply packages/utxo-lib/package.json]: my mistake, should not be added

File: packages/suite-web/package.json (hunk 2 above)
Comment (mroz22) [reply packages/suite-web/package.json:49]: I guess so, cc @andrcj

File: packages/suite-web/e2e/cypress.config.ts (hunk 4 above)
Comment (mroz22) [reply packages/suite-web/e2e/cypress.config.ts:3]: cc @andrcj I don't really maintain this anymore

**Pull request closed without merging**